packages/swarm-docs/src/components/radioExample.js

```diff
@@ -1,35 +1,64 @@
-import React from 'react'
-import { Radio } from '@meetup/swarm-components'
+import React from 'react';
+import { Radio } from '@meetup/swarm-components';
 
 const Example = () => {
-  const [checked, setChecked] = React.useState(false)
-  const [twoChecked, setTwoChecked] = React.useState(false)
+  const [checked, setChecked] = React.useState(false);
+  const [twoChecked, setTwoChecked] = React.useState(false);
+  const [threeChecked, setThreeChecked] = React.useState(false);
+
   return (
     <>
+      <p><b>These are examples of single (not-grouped) Radio buttons:</b></p>
+      <br/>
       <Radio
-        label="swarm"
+        label="Radio button with `label` prop"
         checked={checked}
-        id="example"
         onChange={() => setChecked(!checked)}
+        name="example1"
+        value="1"
       />
+      <br/>
+      <br/>
       <Radio
         checked={twoChecked}
-        id="example2"
         onChange={() => setTwoChecked(!twoChecked)}
-        disabled
+        name="example2"
+        value="2"
       >
-        disabled
+        Radio button using children as label
       </Radio>
+      <br/>
+      <br/>
       <Radio
-        checked
-        id="example2"
-        onChange={() => setTwoChecked(!twoChecked)}
-        disabled
+        checked={threeChecked}
+        onChange={() => setThreeChecked(!threeChecked)}
+        name="example3"
+        value="3"
       >
-        disabled checked
+        <span>Radio Button with <b>nested</b> children as label</span>
       </Radio>
+      <br/>
+      <br/>
+      <Radio
+        label="Disabled radio button"
+        name="example4"
+        value="4"
+        disabled
+      />
+      <br/>
+      <br/>
+      <Radio
+        label="Disabled, checked radio button"
+        name="example5"
+        value="5"
+        disabled
+        checked
+      />
+      <br/>
+      <br/>
+      <p><b>This is an example of how to manage state for a group of radio buttons:</b></p>
```

Comment:
This is a @TODO for the separate ticket, right?

Reply:
Correct!